nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/canvas/nf-parameter-provider.js

```diff
@@ -2383,7 +2439,7 @@
 
                 if (!_.isEmpty(dataContext.parameterStatus.parameter.parameter.referencingComponents)) {
                     // disable checkboxes
-                    if (dataContext.sensitivity === SENSITIVE) {
+                    if (dataContext.sensitivity === SENSITIVE || dataContext.status === 'REMOVED' || dataContext.status === 'MISSING_BUT_REFERENCED') {
```

Comment:
Are these conditions needed? The check above should cover when the status is `REMOVED` or `MISSING_BUT_REFERENCED`.

Reply:
Good catch. No, we don't need the check anymore. It was a leftover change before I decided to check the status before this line. Removed.